src/content/changes/highlights/2021-12-03.md

```diff
@@ -0,0 +1,22 @@
+---
+date: 2021-12-03T10:00:00
+title: Highlights for the week ending December 03, 2021
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Apps
+
+- [aws-load-balancer-controller-app](https://github.com/giantswarm/aws-load-balancer-controller-app) [v1.0.1](https://github.com/giantswarm/aws-load-balancer-controller-app/blob/main/CHANGELOG.md#101---2021-12-02) adds further Giant Swarm monitoring.
+- [kong-app](https://github.com/giantswarm/kong-app/) [v2.3.4](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#234---2021-12-02) adds further Giant Swarm monitoring.
+- [nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app) [v2.5.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#250---2021-11-29) updates the controller image to upstream [v1.1.0](https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.1.0).
+- [nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app) [v2.6.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#260---2021-12-02) adds the ability to specify a custom LoadBalancer Service name suffix.
+
+
+## User interfaces
+ - Version caching is now disabled for the selfupdate command, so that the latest version can be seen right after it’s released.
+ - Make the --release flag mandatory, which specifies the workload cluster release version in the template cluster and template nodepool subcommands.
```

Comment:
I do think this one is important, as the command will fail without the `--release` flag.

```suggestion
 - In kubectl-gs we had to make the `--release` flag mandatory, which specifies the workload cluster release version in the `template cluster` and `template nodepool` subcommands.
```

Reply:
Alright, let me remove it.